tests/src/it/resources/reference.conf

```diff
@@ -18,6 +18,8 @@ akka {
     single-expect-default = 10s
   }
 
+  kafka.testkit.testcontainers.container-logging = true
```

Comment:
I'm a bit surprised we use `reference.conf` in IT tests, shouldn't this be `application.conf`?

Reply:
That confused me as well.  AFAIC it should be `application.conf`.